2019/12/25 20:53:01 [error] 6#6: *3 connect() failed (113: No route to host) while connecting to upstream, client: 206.176.145.183, server: localhost, request: "GET / HTTP/1.1", upstream: "http://172.17.0.3:3000/", host: "140.238.156.136:4000"
2019/12/25 20:53:08 [error] 6#6: *3 connect() failed (113: No route to host) while connecting to upstream, client: 206.176.145.183, server: localhost, request: "GET / HTTP/1.1", upstream: "http://172.17.0.3:3000/", host: "140.238.156.136:4000"
2019/12/25 20:54:39 [error] 6#6: *9 connect() failed (113: No route to host) while connecting to upstream, client: 206.176.145.183, server: localhost, request: "GET / HTTP/1.1", upstream: "http://172.17.0.3:3000/", host: "140.238.156.136:4000"
2019/12/25 20:54:46 [error] 6#6: *9 connect() failed (113: No route to host) while connecting to upstream, client: 206.176.145.183, server: localhost, request: "GET / HTTP/1.1", upstream: "http://172.17.0.3:3000/", host: "140.238.156.136:4000"
2019/12/25 20:56:28 [error] 6#6: *15 connect() failed (113: No route to host) while connecting to upstream, client: 206.176.145.183, server: localhost, request: "GET /status HTTP/1.1", upstream: "http://172.17.0.3:3000/status", host: "140.238.156.136:4000"
2019/12/25 20:57:38 [error] 6#6: *19 connect() failed (113: No route to host) while connecting to upstream, client: 206.176.145.183, server: localhost, request: "GET / HTTP/1.1", upstream: "http://172.17.0.3:3000/", host: "140.238.156.136:4000"
2019/12/25 20:57:51 [error] 6#6: *19 connect() failed (113: No route to host) while connecting to upstream, client: 206.176.145.183, server: localhost, request: "GET /x HTTP/1.1", upstream: "http://172.17.0.3:3000/x", host: "140.238.156.136:4000"
2019/12/25 20:57:58 [error] 6#6: *19 connect() failed (113: No route to host) while connecting to upstream, client: 206.176.145.183, server: localhost, request: "GET /status HTTP/1.1", upstream: "http://172.17.0.3:3000/status", host: "140.238.156.136:4000"
2019/12/25 20:59:36 [emerg] 1#1: unknown directive "vhost_traffic_status_display" in /etc/nginx/conf.d/default_https.conf:19
2019/12/25 21:00:10 [emerg] 1#1: unknown directive "vhost_traffic_status_display" in /etc/nginx/conf.d/default_https.conf:19
2019/12/25 21:11:37 [emerg] 1#1: unknown directive "vhost_traffic_status_zone" in /etc/nginx/nginx.conf:9
2019/12/25 21:21:27 [emerg] 1#1: dlopen() "/etc/nginx/modules/ngx_http_vhost_traffic_status_module.so" failed (/etc/nginx/modules/ngx_http_vhost_traffic_status_module.so: cannot open shared object file: No such file or directory) in /etc/nginx/nginx.conf:7
2019/12/29 20:26:48 [emerg] 1#1: dlopen() "/etc/nginx/modules/ngx_http_vhost_traffic_status_module.so" failed (/etc/nginx/modules/ngx_http_vhost_traffic_status_module.so: cannot open shared object file: No such file or directory) in /etc/nginx/nginx.conf:7
2019/12/29 20:27:20 [emerg] 1#1: dlopen() "/etc/nginx/modules/ngx_http_vhost_traffic_status_module.so" failed (/etc/nginx/modules/ngx_http_vhost_traffic_status_module.so: cannot open shared object file: No such file or directory) in /etc/nginx/nginx.conf:7
2019/12/29 20:42:49 [emerg] 1#1: dlopen() "/etc/nginx/modules/ngx_http_vhost_traffic_status_module.so" failed (/etc/nginx/modules/ngx_http_vhost_traffic_status_module.so: cannot open shared object file: No such file or directory) in /etc/nginx/nginx.conf:7
2019/12/29 20:50:35 [emerg] 1#1: invalid number of arguments in "auth_basic_user_file" directive in /etc/nginx/conf.d/default_https.conf:29
2019/12/29 20:52:17 [warn] 6#6: *7 an upstream response is buffered to a temporary file /var/cache/nginx/proxy_temp/1/00/0000000001 while reading upstream, client: 45.58.95.24, server: localhost, request: "GET / HTTP/1.1", upstream: "http://172.17.0.3:3000/", host: "140.238.156.136:4000"
2019/12/29 20:57:13 [warn] 6#6: *13 an upstream response is buffered to a temporary file /var/cache/nginx/proxy_temp/2/00/0000000002 while reading upstream, client: 45.58.95.24, server: localhost, request: "GET / HTTP/1.1", upstream: "http://172.17.0.3:3000/", host: "140.238.156.136:4000"
2019/12/29 20:57:14 [warn] 6#6: *13 an upstream response is buffered to a temporary file /var/cache/nginx/proxy_temp/3/00/0000000003 while reading upstream, client: 45.58.95.24, server: localhost, request: "GET / HTTP/1.1", upstream: "http://172.17.0.3:3000/", host: "140.238.156.136:4000"
2019/12/29 20:57:15 [warn] 6#6: *25 an upstream response is buffered to a temporary file /var/cache/nginx/proxy_temp/4/00/0000000004 while reading upstream, client: 45.58.95.24, server: localhost, request: "GET / HTTP/1.1", upstream: "http://172.17.0.3:3000/", host: "140.238.156.136:4000"
2019/12/29 20:58:08 [warn] 6#6: *2 an upstream response is buffered to a temporary file /var/cache/nginx/proxy_temp/1/00/0000000001 while reading upstream, client: 45.58.95.24, server: localhost, request: "GET / HTTP/1.1", upstream: "http://172.17.0.3:3000/", host: "140.238.156.136:4000"
2019/12/29 20:59:47 [error] 6#6: *3 access forbidden by rule, client: 45.58.95.24, server: localhost, request: "GET / HTTP/1.1", host: "140.238.156.136:4000"
